typings/index.d.ts

```diff
@@ -2451,7 +2443,7 @@ declare module 'discord.js' {
     RESOURCE_OVERLOADED: 130000;
   }
 
-  type APIMessageContentResolvable = string | number | boolean | bigint | symbol | readonly StringResolvable[];
+  type APIMessageContentResolvable = string | number | boolean | bigint | symbol | readonly string[];
```

Comment:
Shouldn't this type be gone too?
This will currently compile, but fail at runtime:
`message.reply(Symbol.for("hello there"));`